src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -318,6 +324,98 @@ public override IRowCursor[] GetRowCursorSet(out IRowCursorConsolidator consolid
             return cursors;
         }
 
+        public void SaveAsOnnx(OnnxContext ctx)
+        {
+            Host.CheckValue(ctx, nameof(ctx));
+            if (_mapper is ISaveAsOnnx onnx)
+                onnx.SaveAsOnnx(ctx);
```

Comment:
I think elsewhere we *fail* if `CanSaveOnnx` is false, but double check me.

Reply:
This is fine I guess. Alternate would just be to have at the head of this method `Host.Check(CanSaveOnnx)` but that's oK.

---
In reply to: [214433162](https://github.com/dotnet/machinelearning/pull/759#discussion_r214433162) [](ancestors = 214433162)